cmake/KokkosCore_config.h.in

```diff
@@ -26,6 +26,7 @@
 #cmakedefine KOKKOS_ENABLE_MEMKIND
 #cmakedefine KOKKOS_ENABLE_LIBRT
 #cmakedefine KOKKOS_ENABLE_SYCL
+#cmakedefine KOKKOS_IMPL_SYCL_DEVICE_GLOBAL_SUPPORTED
```

Comment:
Nitpick (sorry didn't think of it before) include in `print_configuration` (won't block if you want to do as a follow up)

Reply:
Here you go!